go-controller/pkg/libovsdb/ops/model.go

```diff
@@ -489,15 +490,40 @@ func buildFailOnDuplicateOps(c client.Client, m model.Model) ([]ovsdb.Operation,
 	return c.WhereAny(m, cond).Wait(ovsdb.WaitConditionNotEqual, &timeout, m, field)
 }
 
-// getAllUpdatableFields returns a list of all of the columns/fields that can be updated for a model
-func getAllUpdatableFields(model model.Model) []interface{} {
+func getCustomUpdatableFields(model model.Model, fieldNames []nbdb.LSPCustomField) []interface{} {
+	var fields []interface{}
+	switch t := model.(type) {
+	case *nbdb.LogicalSwitchPort:
+		for _, name := range fieldNames {
+			switch name {
+			case nbdb.LSPCustomFieldEnabled:
+				fields = append(fields, &t.Enabled)
+			default:
+				panic(fmt.Sprintf("getAllUpdatableFieldsByFieldNames: unknown or unsupported field %q for LogicalSwitchPort", name))
+			}
+		}
+	case *nbdb.PortGroup:
```

Comment:
What is the point of this case? it will always panic same as default.

Reply:
this function was removed.